src/app/api/password-reset/route.ts

```diff
@@ -16,35 +16,29 @@ export const POST = withAppRouterHighlight(async (request: NextRequest) => {
     });
   }
 
-  try {
-    const storedUser = await db
-      .selectFrom("user")
-      .selectAll()
-      .where("email", "=", parsedEmail.data)
-      .executeTakeFirst();
-
-    if (!storedUser) {
-      return new Response(
-        "A password reset email will be sent if you have a Flojoy Cloud account. Please check your inbox :)",
-        {
-          status: 200,
-        },
-      );
-    }
-
-    const token = await createPasswordResetToken(storedUser.id);
-    const resetLink = `${env.NEXT_PUBLIC_URL_ORIGIN}/password-reset/${token}`;
-    await sendPasswordResetLink(storedUser.email, resetLink);
+  const storedUser = await db
+    .selectFrom("user")
+    .selectAll()
+    .where("email", "=", parsedEmail.data)
+    .executeTakeFirst();
 
+  if (!storedUser) {
     return new Response(
       "A password reset email will be sent if you have a Flojoy Cloud account. Please check your inbox :)",
       {
         status: 200,
       },
     );
-  } catch (e) {
-    return new Response(String(e), {
-      status: 400,
-    });
   }
+
+  const token = await createPasswordResetToken(storedUser.id);
+  const resetLink = `${env.NEXT_PUBLIC_URL_ORIGIN}/password-reset/${token}`;
+  await sendPasswordResetLink(storedUser.email, resetLink);
+
+  return new Response(
+    "A password reset email will be sent if you have a Flojoy Cloud account. Please check your inbox :)",
```

Comment:
Could u extract to a const string such that people don't accidently update one of them and forgot to update the other.

Reply:
Sure, thanks!